cocos/core/scene-graph/scene.ts

```diff
@@ -150,156 +133,28 @@ export class Scene extends BaseNode {
-
-    /**
-     * @en
-     * Refer to [[Node.getScale]]
-     * @zh
-     * 参考 [[Node.getScale]]
-     */
-    public getScale (out?: Vec3): Vec3 { return Vec3.copy(out || new Vec3(), Vec3.ONE); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldPosition]]
-     * @zh
-     * 参考 [[Node.getWorldPosition]]
-     */
-    public getWorldPosition (out?: Vec3) { return Vec3.copy(out || new Vec3(), Vec3.ZERO); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldRotation]]
-     * @zh
-     * 参考 [[Node.getWorldRotation]]
-     */
-    public getWorldRotation (out?: Quat) { return Quat.copy(out || new Quat(), Quat.IDENTITY); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldScale]]
-     * @zh
-     * 参考 [[Node.getWorldScale]]
-     */
-    public getWorldScale (out?: Vec3) { return Vec3.copy(out || new Vec3(), Vec3.ONE); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldMatrix]]
-     * @zh
-     * 参考 [[Node.getWorldMatrix]]
-     */
-    public getWorldMatrix (out?: Mat4): Mat4 { return Mat4.copy(out || new Mat4(), Mat4.IDENTITY); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldRS]]
-     * @zh
-     * 参考 [[Node.getWorldRS]]
-     */
-    public getWorldRS (out?: Mat4): Mat4 { return Mat4.copy(out || new Mat4(), Mat4.IDENTITY); }
-
-    /**
-     * @en
-     * Refer to [[Node.getWorldRT]]
-     * @zh
-     * 参考 [[Node.getWorldRT]]
-     */
-    public getWorldRT (out?: Mat4): Mat4 { return Mat4.copy(out || new Mat4(), Mat4.IDENTITY); }
-
-    /**
-     * @en
-     * Refer to [[Node.position]]
-     * @zh
-     * 参考 [[Node.position]]
-     */
-    public get position (): Readonly<Vec3> { return Vec3.ZERO; }
-
-    /**
-     * @en
-     * Refer to [[Node.worldPosition]]
-     * @zh
-     * 参考 [[Node.worldPosition]]
-     */
-    public get worldPosition (): Readonly<Vec3> { return Vec3.ZERO; }
-
-    /**
-     * @en
-     * Refer to [[Node.rotation]]
-     * @zh
-     * 参考 [[Node.rotation]]
-     */
-    public get rotation (): Readonly<Quat> { return Quat.IDENTITY; }
-
-    /**
-     * @en
-     * Refer to [[Node.worldRotation]]
-     * @zh
-     * 参考 [[Node.worldRotation]]
-     */
-    public get worldRotation (): Readonly<Quat> { return Quat.IDENTITY; }
-
-    /**
-     * @en
-     * Refer to [[Node.scale]]
-     * @zh
-     * 参考 [[Node.scale]]
-     */
-    public get scale (): Readonly<Vec3> { return Vec3.ONE; }
-
-    /**
-     * @en
-     * Refer to [[Node.worldScale]]
-     * @zh
-     * 参考 [[Node.worldScale]]
-     */
-    public get worldScale (): Readonly<Vec3> { return Vec3.ONE; }
-
-    /**
-     * @en
-     * Refer to [[Node.eulerAngles]]
-     * @zh
-     * 参考 [[Node.eulerAngles]]
-     */
-    public get eulerAngles (): Readonly<Vec3> { return Vec3.ZERO; }
-
-    /**
-     * @en
-     * Refer to [[Node.worldMatrix]]
-     * @zh
-     * 参考 [[Node.worldMatrix]]
-     */
-    public get worldMatrix (): Readonly<Mat4> { return Mat4.IDENTITY; }
```

Comment:
Why remove these functions?

Reply:
Now can directly use parent functions ([Node.worldMatrix]). And those Readonly functions will get errors when editor init

Reply:
Yes. They are related.